packages/headless/test/fixtures/reinstall-peer-deps/package.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "reinstall-peer-deps",
+  "version": "1.0.0",
+  "devDependencies": {
+    "eslint-config-standard": "17.0.0"
```

Comment:
I don’t have confidence about how to write dependencies for the test, so if there is something to improvement, I would be happy if you point out.

Reply:
We use custom packages stored in https://github.com/pnpm/registry-mock for e2e. Those packages are small and faster to install, so tests are fast.